include/assert.h

```diff
@@ -1,6 +1,16 @@
 #ifndef _ASSERT_H_
 #define _ASSERT_H_
 
+#if TARGET_PC
```

Comment:
The standard C library headers here in the include directory are intended to only be used for Gamecube. Can you configure CMake to use the system's headers instead?